python/torch_mlir/extras/fx_importer.py

```diff
@@ -507,12 +515,12 @@ def node_val_to_type(self, node: torch_fx.Node) -> IrType:
                         f"Quantized tensor meta data is not supported."
                     )
                 else:
-                    return self.tensor_metadata_to_type(tensor_meta, sparse_layout)
+                    return self.tensor_metadata_to_type(tensor_meta, sparsity)
             elif val is not None:
                 # some nodes with symbolic inputs pass a 'val' attribute rather than
                 # tensor_meta
                 if isinstance(val, TorchFakeTensor):
-                    return self.get_vtensor_type(val.size(), val.dtype, sparse_layout)
+                    return self.get_vtensor_type(val.size(), val.dtype, sparsity)
```

Comment:
Will pass None for sparsity to get_vtensor_type, which I think will trigger and UnboundLocalError.

Reply:
sparsity is None is accounted for?